database/migrations/20221201070519-user-migrations.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable no-unused-vars */
+/** @type {import('sequelize-cli').Migration} */
```

Comment:
@jonyoung123 what's this for

Reply:
automatically generated from sequelize cli commands